dev-util/cgvg/cgvg-1.6.3.ebuild

```diff
@@ -0,0 +1,15 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="minimal command-line source browsing tool similar to cscope"
```

Comment:
Let's start with a capital letter, then we're good to merge. ;)

Reply:
hehe, sure :)